### PR TITLE
Add explicit profile s3 config example

### PR DESCRIPTION
**Pull request by joguSD**

### CODE REVIEW

File: awscli/topics/s3-config.rst [hunk 1]
```diff
@@ -82,6 +82,12 @@ could instead run these commands::
     $ aws configure set default.s3.use_accelerate_endpoint true
     $ aws configure set default.s3.addressing_style path
 
+To programmatically set these values for a profile other than the default
```
Comment (vz10) [inline awscli/topics/s3-config.rst:85]: In sounds a bit confusing as for me, probably `programmatically`  is redundant here.

**Review by vz10 — COMMENTED** [review]
Looks good, just a small suggestion

File: awscli/topics/s3-config.rst (hunk 1 above)
Comment (joguSD) [reply awscli/topics/s3-config.rst:85]: I used this wording the be consistent with the previous paragraph.

**Review by kyleknap — APPROVED** [review]
Looks good. :ship:

**Pull request merged**